cli/azd/test/functional/cli_test.go

```diff
@@ -66,8 +66,14 @@ func Test_CLI_Version_PrintsVersion(t *testing.T) {
 
 	rn := os.Getenv("GITHUB_RUN_NUMBER")
 	if rn != "" {
-		// TODO: This should be read from cli/version.txt (https://github.com/Azure/azure-dev/issues/36)
-		require.Contains(t, out, "0.2.0")
+		_, filename, _, gotFilePath := runtime.Caller(0)
+		require.True(t, gotFilePath)
+		runningTestPath := filepath.Dir(filename)
+		require.Contains(t, runningTestPath, "cli")
+		cliPath := strings.Split(runningTestPath, "cli")[0]
+		version, err := os.ReadFile(path.Join(cliPath, "cli/version.txt"))
```

Comment:
```suggestion
		version, err := os.ReadFile(filepath.Join(cliPath, "cli", "version.txt"))
```

Reply:
Othe other option would be to consider using GetAzdLocation() and work of off that - but this seems perfectly fine as well.

Reply:
GetAzdLocation() calls `runtime.Caller(0)`

I have changed it to re-used GetAzdLocation() so we keep that runtime call in one place only. Thank you!